asf_tools/vector.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+from typing import Iterator, List, Union
+
+from osgeo import ogr
+
+ogr.UseExceptions()
+
+
+def get_features(vector_path: Union[str, Path]) -> List[ogr.Feature]:
+    ds = ogr.Open(str(vector_path))
+    layer = ds.GetLayer()
+    return [feature for feature in layer]
+
+
+def intersects_features(geometry: ogr.Geometry, features: Iterator) -> bool:
+    for feature in features:
+        if feature.GetGeometryRef().Intersects(geometry):
+            return True
+
+
+def intersecting_feature_properties(geometry: ogr.Geometry, features: Iterator,
```

Comment:
not in love with this function name, but don't have a great alternative yet, something like `get_property_values_for_intersecting_features` but that's too long

Reply:
is it?